src/SFA.DAS.EmployerIncentives.Commands/ApprenticeshipIncentive/CreatePayment/CreatePaymentCommandValidator.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Threading.Tasks;
+using SFA.DAS.EmployerIncentives.Abstractions.Commands;
+
+namespace SFA.DAS.EmployerIncentives.Commands.ApprenticeshipIncentive.CreatePayment
```

Comment:
We don't need this.

In the AddCommandHandlerDecorators extension method add this

.AddSingleton(typeof(IValidator<CreatePaymentCommand>), new NullValidator())

and delete this file.

Reply:
Nice. Will add that to the EI-51 branch.